client/packages/purchasing/src/goods_received/DetailView/LineEdit/GoodsReceivedLineEditModal.tsx

```diff
@@ -0,0 +1,86 @@
+import { useDialog, useNotification } from '@common/hooks';
+import { DialogButton, InlineSpinner } from '@common/components';
+import { useTranslation, Box } from '@openmsupply-client/common';
+import React, { useState } from 'react';
+import { GoodsReceivedLineEdit } from './GoodsReceivedLineEdit';
+import { GoodsReceivedFragment } from '../../api/operations.generated';
+import { useGoodsReceivedLine } from '../../api';
+
+interface GoodsReceivedLineEditModalProps {
+  lineId: string | null;
+  goodsReceived: GoodsReceivedFragment;
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export const GoodsReceivedLineEditModal = ({
+  lineId,
+  goodsReceived,
+  isOpen,
+  onClose,
+}: GoodsReceivedLineEditModalProps) => {
+  const t = useTranslation();
+  const { error } = useNotification();
+
+  const lines = goodsReceived.lines.nodes;
+
+  const [currentLine, setCurrentLine] = useState(
+    lines.find(line => line.id === lineId) ?? undefined
+  );
+
+  const isUpdating = false; // remove me when adding update
+  const { draft, updatePatch } = useGoodsReceivedLine(currentLine?.id);
+
+  const handleSave = async () => {
+    try {
+      // await update();
+      alert('Not implemented yet!');
+      return true;
+    } catch (e: unknown) {
+      if (e instanceof Error) error(e.message)();
+      else error('unknown error')();
+      return false;
+    }
+  };
```

Comment:
Have remove create functionalities as AddButton and PurchaseOrderLineSearchModal is responsible for adding a new PO line to a Goods Received